package.json

```diff
@@ -12,12 +12,14 @@
     "pretest": "lerna run pretest",
     "test": "node scripts/test.js",
     "posttest": "lerna run posttest",
-    "test:watch": "node scripts/test.js --watch",
+    "test:watch": "run-script-os",
+    "test:watch:darwin:linux": "sleep 10 && yarn test:watch:default",
+    "test:watch:default": "node scripts/test.js --watch",
     "lint": "tsc -b && yarn prettier:check && eslint --ext .js,.jsx,.ts,.tsx,.graphql .",
-    "lint:fix": "eslint --ext .js,.jsx,.ts,.tsx,.graphql . --fix; prettier --ignore-path .eslintignore --write '**/*.{js,jsx,ts,tsx,graphql,md}'",
-    "prettier:check": "prettier --ignore-path .eslintignore --check '**/*.{js,jsx,ts,tsx,graphql,md}'",
+    "lint:fix": "eslint --ext .js,.jsx,.ts,.tsx,.graphql . --fix && prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,ts,tsx,graphql,md}\"",
+    "prettier:check": "prettier --ignore-path .eslintignore --check \"**/*.{js,jsx,ts,tsx,graphql,md}\"",
     "depcheck": "lerna exec --stream 'yarn depcheck --ignores=\"@app/config,@app/client,tslib,webpack,babel-plugin-import,source-map-support,@graphql-codegen/*,graphql-toolkit,net,tls,dayjs\" --ignore-dirs=\".next\"'",
-    "dev": "yarn && lerna run codegen --stream && tsc -b && concurrently --kill-others --names 'TSC,WATCH,RUN,TEST' --prefix '({name})' --prefix-colors 'yellow.bold,yellow.bold,cyan.bold,greenBright.bold' 'tsc -b --watch --preserveWatchOutput --assumeChangesOnlyAffectDirectDependencies' 'lerna run --parallel watch' 'lerna run --parallel dev' 'sleep 10; yarn test:watch'",
+    "dev": "yarn && lerna run codegen --stream && tsc -b && concurrently -k -n \"TSC,WATCH,RUN,TEST\" -p \"({name})\" -c \"yellow.bold,yellow.bold,cyan.bold,greenBright.bold\" \"tsc -b --watch --preserveWatchOutput\" \"lerna run --parallel watch\" \"lerna run --parallel dev\" \"yarn test:watch\"",
```

Comment:
Please use expanded option flags, it makes it clearer to people what the flags do when they're not familiar with the tool.

What's the reason we can't use single quotes here; does Windows not support them? Bit frustrating to have to backslash escape everything.

Reply:
Yes, I agree it is a bit cumbersome to add escaping, unfortunately it is the only way Windows supports those commands right now.